libraries/Microsoft.Bot.Connector.Streaming/Transport/StreamingTransport.cs

```diff
@@ -191,6 +196,8 @@ private async Task StartReceivingAsync(CancellationToken cancellationToken)
                     await _application.Output.CompleteAsync(ex).ConfigureAwait(false);
                     Log.TransportError(Logger, ex);
                 }
+
+                throw;
```

Comment:
On further thinking about this change, I think this throw will affect all streaming exchanges (NamedPipes and WebSockets) with the bot. This may have side effects to web socket connections. How about keeping the changes within CloudAdapter, and just adding a retry block around the code in `ConnectNamedPipeAsync`? For DL-ASE, we expect the named pipe connection to stay alive as long as the app service is running. So, we could just retry the connection when it disconnects. Please let me know your thoughts.

Reply:
Hi @msomanathan, we changed the fix to keep the connection alive updating only the CloudAdapter class.
Thanks!